syntax = "proto3";

package simple;

message HelloMessage {
  string name = 1;
  string msg = 2;
}

message ReplyMessage {
  string reply_msg = 1;
}

service SimpleService {
  rpc HelloServer(stream HelloMessage) returns (stream ReplyMessage) {}
}